projects/packages/forms/src/dashboard/inbox/export-responses/index.js

```diff
@@ -68,7 +69,12 @@ const ExportResponsesButton = () => {
 
 	return (
 		<>
-			<Button className="export-button" variant="primary" onClick={ openModal }>
+			<Button
```

Comment:
The export button is misaligned. Is this intentional?

Reply:
Can you clarify what the alignment issue is you're seeing?

Reply:
<img width="816" alt="Screenshot 2025-05-06 at 9 44 08 AM" src="https://github.com/user-attachments/assets/4b11fab5-f23e-45c7-a051-39b6af8ef490" />

I'd expect the export button to be at the edge of the container, similar to `view` button.

Reply:
Yes, this is being updated in a separate PR now here: 
https://github.com/Automattic/jetpack/pull/43381